code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -284,7 +284,7 @@ AI MODULES
 	name = "'NT Default' Core AI Module"
```

Comment:
Shouldn't this be renamed to "Eris Default"?

Reply:
May be `/obj/item/weapon/aiModule` should rebuild its name in `Initialize`